Header header
string group_name
int8 is_valid
int64 waypoint_count
int64 fail_index
float32[] distances

